### ISSUE TITLE
Elevated urinary prostaglandin E2 level

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7169]
**Preferred term label:**

- Also Elevated urinary PGE-M level
**Synonyms**
Elevated urinary PGE2 level

**Definition (free text, please give PubMed ID)**
An increased amount of prostaglandin E2 in the urine.

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
167100  

PMID: 24838973  3/3 (MALE only)

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
